setup.py

```diff
@@ -50,6 +50,7 @@ def read(*rnames):
           'Programming Language :: Python :: 3.10',
           'Programming Language :: Python :: 3.11',
           'Programming Language :: Python :: 3.12',
+          'Programming Language :: Python :: 3.13',
```

Comment:
We still declare `Python :: 3.7` support.

Reply:
Thanks good catch, that was an oversight on my part. I just removed it.